package.json

```diff
@@ -14,13 +14,14 @@
   },
   "scripts": {
     "test": "jest",
+    "watch-tests": "jest --watch",
     "prepublish": "yarn run build",
     "build": "mkdir -p lib && babel index.js --out-file lib/index.js"
   },
   "dependencies": {
-    "browserslist": "^4.0.1",
+    "browserslist": "^4.3.4",
```

Comment:
This version of browserslist knows that Safari 12 is a thing, whereas browserslist@4.0.1 does not